msprime/simulations.py

```diff
@@ -26,6 +26,8 @@
 import random
 import sys
 import os
+import scipy.linalg
```

Comment:
Note that this brings in scipy as a prerequisite. We could write our own expm function (not too bad for this nicely behaved case) to avoid this.